spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -323,6 +323,38 @@ def get_rating_summary(app_id, platform, versionId = '')
       parse_response(r, 'data')
     end
 
+    def create_developer_response!(app_id: nil, platform: "ios", review_id: nil, response: nil)
+      raise "app_id is required" unless app_id
+      raise "review_id is required" unless review_id
+      raise "response is required" unless response
```

Comment:
👍 on asserting the required parameters.

Perhaps we can move https://github.com/fastlane/fastlane/blob/master/spaceship/lib/spaceship/test_flight/client.rb#L191-L211 into `Spaceship::Client` so that it can be re-used here.